srcpkgs/dkms/files/kernel.d/dkms.prerm

```diff
@@ -6,9 +6,9 @@ VERSION="$2"
 
 if [ -x /usr/bin/dkms ]; then
 while read line; do
-   name=`echo "$line" | awk '{print $1}' | sed 's/,$//'`
-   vers=`echo "$line" | awk '{print $2}' | sed 's/,$//'`
-   arch=`echo "$line" | awk '{print $4}' | sed 's/:$//'`
+   name=`echo "$line" | awk '{print $1}' | sed 's/,$//' | cut -d'/' -f1`
+   vers=`echo "$line" | awk '{print $1}' | sed 's/,$//' | cut -d'/' -f2`
+   arch=`echo "$line" | awk '{print $3}' | sed 's/:$//'`
```

Comment:
Same here. And why change permissions?

Reply:
> And why change permissions?

Oops. I think it might be my editor's fault.